aptoide-database/src/main/java/cm/aptoide/pt/database/room/RoomUpdate.java

```diff
@@ -269,27 +273,31 @@ public void setHasAppc(boolean hasAppc) {
     this.hasAppc = hasAppc;
   }
 
-  public RealmList<Split> getSplits() {
-    return splits;
+  public List<RoomSplit> getRoomSplits() {
+    return roomSplits;
   }
 
-  public void setSplits(RealmList<Split> splits) {
-    this.splits = splits;
+  public void setRoomSplits(List<RoomSplit> roomSplits) {
+    this.roomSplits = roomSplits;
   }
 
-  public RealmList<RealmString> getRequiredSplits() {
+  public List<String> getRequiredSplits() {
     return requiredSplits;
   }
 
-  public void setRequiredSplits(RealmList<RealmString> requiredSplits) {
+  public void setRequiredSplits(List<String> requiredSplits) {
     this.requiredSplits = requiredSplits;
   }
 
   public boolean hasSplits() {
-    return splits != null && !splits.isEmpty();
+    return roomSplits != null && !roomSplits.isEmpty();
   }
 
   public String getStoreName() {
     return storeName;
   }
+
+  public void setStoreName(String storeName) {
+    this.storeName = storeName;
+  }
```

Comment:
room needs setters for all attributes of the table

Reply:
that is only true for params non being passed in the constructor. storeName is always pass in the all-args constructor, if the issue is having an empty constructor, we can remove it since is not being used anyway.